lib/identity/client.js

```diff
@@ -886,16 +926,99 @@ class Client extends PartialClient {
       )
     }
     // if the admin sharing group will be empty, revoke the share & delete the group
-    const groupName = `tozny.secret.${this.config.realmName}.${
-      this.storage.config.clientId
-    }.${secret.meta.plain.secretType.toLowerCase()}`
+    const groupName = `tozny.secret.${this.storage.config.clientId}.${this.config.realmName}`
     await this.revokeShareBeforeDeleteSecret(groupName, secret.meta.type)
     // delete the secret
     return await this.storage.deleteRecord(
       secret.meta.recordId,
       secret.meta.version
     )
   }
+  /**
+   * isValidToznySecretNamespace
+   * @param {Object} groupName
+   * @return {Boolean} returns true if the secret was successfully removed
+   */
+  async isValidToznySecretNamespace(groupName) {
```

Comment:
This is probably not useful in the client context (it's not something I would ever expect a user to want to call) for that reason, I'd like to move it to a utility function instead.